en/docs/how-tos/endpoints/connect-to-an-endpoint-via-a-proxy.md

```diff
@@ -0,0 +1,70 @@
+# Connect to an Endpoint via a Proxy
+
+There can be certain scenarios where microgateway has to connect to the backend service via a proxy server. This can due to the reason that microgateway sits behind a firewall
```

Comment:
```suggestion
There can be certain scenarios where Microgateway has to connect to the backend service via a proxy server. This can be due to the reason that Microgateway sits behind a firewall.
```